internal/venice-consumer/src/main/java/com/linkedin/venice/client/consumer/VeniceChangelogConsumer.java

```diff
@@ -0,0 +1,63 @@
+package com.linkedin.venice.client.consumer;
+
+import com.linkedin.venice.annotation.Experimental;
+import com.linkedin.venice.pubsub.api.PubSubMessage;
+import java.util.Collection;
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+
+
+/**
+ * Venice change capture consumer to provide
```

Comment:
the sentence doesn't sound complete
I think we should either add something after `to provide` or just keep `A change capture consumer`
Also, add descriptions of `K` and `V`. And why do we need them?

Reply:
Will add it